forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -138,9 +86,9 @@ def generate_exhaustive_process_dfe_experiment(program: Program, qubits: list,
         used in ``program``.
     :param benchmarker: A ``BecnhmarkConnection`` object to be used in experiment design
     :return: a set of experiments
-    :rtype: ``TomographyExperiment`
+    :rtype: ``ObservablesExperiment`
```

Comment:
Can just drop the `rtype` part of the docs, since it already indicated by the type annotation on the function signature.